app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -15,8 +15,7 @@
   <div class="col-12 sm-col-5 mb2 sm-mb0 sm-mr-20p inline-block">
     <%= text_field_tag(:code, '', value: @presenter.code_value, required: true, autofocus: true,
                        pattern: '[0-9]*', class: 'col-12 field monospace mfa',
-                       'aria-describedby': 'code-instructs', maxlength: Devise.direct_otp_length,
-                       type: 'number') %>
```

Comment:
Cleave doesn't like `type: 'number'` here, as demonstrated by some js console errors. Cleave likes to format 'text' and not 'numbers' or something like that. So, I just restricted to numerals only in the Cleave formatter

Reply:
Some of those are used for mobile phones to make sure that the phones only show the numeric keyboard right? Have you tried this on a phone?

Reply:
Ooh, good point. I saw a suggestion online to use 'tel' here, maybe I'll try that